OsmAnd/src/net/osmand/plus/exploreplaces/ExplorePlacesFragment.kt

```diff
@@ -55,8 +57,9 @@ class ExplorePlacesFragment : BaseOsmAndFragment(), NearbyItemClickListener,
 	private val singleThreadExecutor = Executors.newSingleThreadExecutor()
 	private var convertAmenitiesTask: ConvertAmenitiesTask? = null
 
-	private lateinit var poiUIFilter: PoiUIFilter
-	private lateinit var adapter: ExplorePlacesAdapter
+	private var poiUIFilter: PoiUIFilter? = null
+	private var poiUIFilterID: String? = null
```

Comment:
Do we really need separate field poiUIFilterID? why not get it from poiUIFilter & savedInstanceState?